### ISSUE TITLE
Failed import 

### ISSUE THREAD

**Issue by InesPM** [FRBs/pygedm#7]
After installation, I get an error when trying to import:
```

  File "scat_scint.py", line 11, in <module>
    import pygedm
  File "/home/ines/anaconda3/lib/python3.7/site-packages/pygedm/__init__.py", line 7, in <module>
    from . import ne2001_wrapper
  File "/home/ines/anaconda3/lib/python3.7/site-packages/pygedm/ne2001_wrapper.py", line 5, in <module>
    from . import dmdsm     # f2py FORTRAN object
ImportError: cannot import name 'dmdsm' from 'pygedm' (/home/ines/anaconda3/lib/python3.7/site-packages/pygedm/__init__.py)

```

**Comment by telegraphic** [comment]
ok, I can recreate this, looking into it!

**Comment by telegraphic** [comment]
Looks like it might be specific to OSX Big Sur (I hope you're using a Mac otherwise I have no idea!). After reading [a completely unrelated project's similar issue](https://github.com/ponylang/ponyc/issues/3684), the solution (for me at least) was to run:

```
xcode-select --install
```

The error message gets hidden by pip. Here's the underlying issue:

```
/usr/local/bin/gfortran -Wall -g -arch x86_64 -Wall -g -undefined dynamic_lookup -bundle /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/src.macosx-10.9-x86_64-3.9/densitymodule.o /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/src.macosx-10.9-x86_64-3.9/fortranobject.o /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/density.NE2001.o /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/src.macosx-10.9-x86_64-3.9/density-f2pywrappers.o -L./ -L/usr/local/Cellar/gcc/10.2.0_3/lib/gcc/10/gcc/x86_64-apple-darwin20/10.2.0 -L/usr/local/Cellar/gcc/10.2.0_3/lib/gcc/10/gcc/x86_64-apple-darwin20/10.2.0/../../.. -L/usr/local/Cellar/gcc/10.2.0_3/lib/gcc/10/gcc/x86_64-apple-darwin20/10.2.0/../../.. -lNE2001 -lgfortran -o ./density.cpython-39-darwin.so
ld: library not found for -lSystem
collect2: error: ld returned 1 exit status
error: Command "/usr/local/bin/gfortran -Wall -g -arch x86_64 -Wall -g -undefined dynamic_lookup -bundle /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/src.macosx-10.9-x86_64-3.9/densitymodule.o /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/src.macosx-10.9-x86_64-3.9/fortranobject.o /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/density.NE2001.o /var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/var/folders/48/17k0cnx11j70zwdp5vc4xy5w0000gp/T/tmpypmjsawb/src.macosx-10.9-x86_64-3.9/density-f2pywrappers.o -L./ -L/usr/local/Cellar/gcc/10.2.0_3/lib/gcc/10/gcc/x86_64-apple-darwin20/10.2.0 -L/usr/local/Cellar/gcc/10.2.0_3/lib/gcc/10/gcc/x86_64-apple-darwin20/10.2.0/../../.. -L/usr/local/Cellar/gcc/10.2.0_3/lib/gcc/10/gcc/x86_64-apple-darwin20/10.2.0/../../.. -lNE2001 -lgfortran -o ./density.cpython-39-darwin.so" failed with exit status 1
```

That is, we get a `ld: library not found for -lSystem`, because it can't find something from xcode.

**Comment by abatten** [comment]
I am getting a similar error on Github Actions when trying to test using `pygedm` with `fruitbat`. However that test is only using `Ubuntu-latest`.

`ImportError: cannot import name 'dmdsm' from partially initialized module 'pygedm' (most likely due to a circular import)`

e.g see https://github.com/abatten/fruitbat/runs/2621302889?check_suite_focus=true

**Comment by telegraphic** [comment]
This one is fixed in 3.2.0 -- closing!
